test/passing/doc_comments-after.ml.ref

```diff
@@ -232,8 +232,7 @@ end = struct
   (* [@@@some attribute] *)
   (* (** Attribute *) *)
 
-  (** Extension *)[%%some
-  extension]
+  (** Extension *)[%%some extension]
```

Comment:
Not related but there should be a break after the doc comment.

Reply:
It can indeed be fixed in another PR.